frontend/src/app/commonComponents/TestResultGuidance/RsvResultGuidance.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
```

Comment:
⚠️ most of the content in this file was taken from MultiplexResultsGuidance.tsx which is now deleted